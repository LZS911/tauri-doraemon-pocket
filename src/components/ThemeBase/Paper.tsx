import { IPaperProps } from '.';

const Paper: React.FC<IPaperProps> = ({ children, className = '' }) => {
  return (
    <div className={`bg-white dark:bg-darkMode dark:text-white ${className}`}>
      {children}
    </div>
  );
};

export default Paper;
