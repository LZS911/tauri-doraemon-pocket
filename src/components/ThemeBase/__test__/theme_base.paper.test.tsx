import { render } from '@testing-library/react';
import Paper from '../Paper';

describe('test Paper', () => {
  test('should match snapshot', () => {
    const { container } = render(
      <Paper>
        <div>children</div>
      </Paper>
    );

    expect(container).toMatchSnapshot();
  });
});
