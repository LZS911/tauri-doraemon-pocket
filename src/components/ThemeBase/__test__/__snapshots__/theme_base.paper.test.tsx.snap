// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Paper should match snapshot 1`] = `
<div>
  <div
    class="bg-white dark:bg-darkMode dark:text-white "
  >
    <div>
      children
    </div>
  </div>
</div>
`;
